test/dataframe.jl

```diff
@@ -179,7 +179,7 @@ module TestDataFrame
         @test size(df, 2) == 5
         @test typeof(df[:, 1]) == Vector{Float64}
 
-        df = convert(DataFrame, Matrix{Float64}(I, 10, 5))
+        df = convert(DataFrame, Matrix{Float64}(undef, 10, 5))
```

Comment:
The `I` there was intentional, as previously this was using `eye`.

Reply:
Is there any way to get that in 1.0? I just realized we weren't actually testing any `I`-specific values, so 🤷‍♂️

Reply:
Yeah `Matrix{Float64}(I, 10, 5)` is how you get it in 1.0, you just need `using LinearAlgebra` to get `I`.